api/src/main/java/de/valtech/aecu/api/service/AecuMigrationService.java

```diff
@@ -0,0 +1,10 @@
+package de.valtech.aecu.api.service;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * An OSGi service that executes an AECU migration based on the AEM node store type on activation
+ */
+@ProviderType
+public interface AecuMigrationService {
```

Comment:
The empty interface does not bring too much value. Please delete and rename AecuMigrationServiceImpl to AecuMigrationService.

Reply:
It was mostly for possible future changes or exposing of the migration service through the API.

Reply:
@gruberrolandvaltech do you want the one class (after removing the interface) in `api` or `core`?

Reply:
It should be one class in core. We can add an interface later if needed.